src/worker/lambda/handlerPuller.go

```diff
@@ -26,6 +26,7 @@ type HandlerPuller struct {
 	prefix   string   // combine with name to get file path or URL
 	dirCache sync.Map // key=lambda name, value=version, directory path
 	dirMaker *common.DirMaker
+	compiled *regexp.Regexp
```

Comment:
Better name possible to indicate what it mattches?  What about something like handlerNameRegex?